dist/settings.json

```diff
@@ -3,9 +3,7 @@
     "Version": "3.0.0",
     "AnnounceInterval": 120,
     "KeepAllItemsOnDeath": false,
-    "KeptItemsOnDeath": [],
-    "LocalChatOverHead": true,
-    "LocalChatInChat": false
+    "KeptItemsOnDeath": []
```

Comment:
Why was his removed?

Reply:
I dont even know

Reply:
I dont think I removed it, I think it was a problem with conflicted local and git versions

Reply:
I added it back